docker-compose.yml

```diff
@@ -332,4 +332,29 @@ services:
     ulimits:
       nofile:
         soft: 262144
-        hard: 262144
\ No newline at end of file
+        hard: 262144
+
+
+# Web-client configuration
+  web-client:
+    image: gokulr03/hyperswitch-web:latest # Need to enter hyperswitch-web-image 
```

Comment:
When I said "build the image ourselves", I meant to include a Dockerfile for the same, so that that Docker image can be built when the web client is brought up. I didn't mean that you had to build an image and push it to Docker Hub.